doc/source/whatsnew/v1.2.0.rst

```diff
@@ -247,6 +247,7 @@ Numeric
 ^^^^^^^
 - Bug in :func:`to_numeric` where float precision was incorrect (:issue:`31364`)
 - Bug in :meth:`DataFrame.any` with ``axis=1`` and ``bool_only=True`` ignoring the ``bool_only`` keyword (:issue:`32432`)
+- Fixed Series.value_counts so that normalize excludes NA values when dropna=False. (:issue:`25970`)
```

Comment:
can you change so that
``` :meth:`Series.value_counts` ``` otherwise won't render, also put dropna=False in double-backtics.